src/components/WeeklySchedule.tsx

```
@@ -141,33 +145,58 @@ export default function WeeklySchedule({ tasks, onTaskSelect, onTodoUpdate, sele
+    })
     
     // すべてのTODOを一度に処理し、適切な日付に配置
-    tasks.forEach(task => {
+    filteredTasks.forEach(task => {
       task.todos.forEach(todo => {
+        // TODO自体の担当者でもフィルタリング
+        const todoAssigneeIds = todo.assigneeIds || task.assigneeIds
+        const isAssignedToSelectedUser = todoAssigneeIds && todoAssigneeIds.some(id => selectedUserIds.includes(id))
+        const isUnassigned = !todoAssigneeIds || todoAssigneeIds.length === 0
+        
+        // フィルタリング条件に合わない場合はスキップ
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)